src/components/button-minor/button-minor.style.ts

```diff
@@ -17,6 +17,16 @@ function makeColors(color: string) {
 }
 
 const StyledButtonMinor = styled(Button)`
+  ${({ iconType, iconTooltipMessage, children }) =>
+    iconType &&
+    iconTooltipMessage &&
```

Comment:
I don't think we need all these conditions.

I see that we need to check for an icon-only button (with the `!children` check), otherwise the icon overlaps the text when both are present. But `iconTooltipMessage` makes no difference here (as far as I can tell), and `iconType` isn't needed simply because without that there's no `StyledIcon` inside for the styles to affect.

I prefer it checking just this one condition as it makes the code simpler with fewer special cases.